front/app/containers/Admin/projects/project/files/components/FeatureInformation/index.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+
+import {
+  Box,
+  colors,
+  Divider,
+  stylingConsts,
+} from '@citizenlab/cl2-component-library';
+
+import FeatureInformationItem from './FeatureInformationItem';
+import messages from './messages';
+
+const FeatureInformation = () => {
+  return (
+    <Box
+      p="20px"
+      display="flex"
+      flexDirection="column"
+      gap="20px"
+      maxWidth="600px"
+      background={colors.white}
+      borderRadius={stylingConsts.borderRadius}
+    >
+      <FeatureInformationItem
+        iconName={'upload-file'}
```

Comment:
Minor: For consistence, let's use
```suggestion
        iconName="upload-file"
```

Add apply the same to the other places in this file. Other `iconNames`